README.md

```diff
@@ -68,6 +68,40 @@ Where:
 * `DOJO_URL` is the URL where DefectDojo is installed.
 * `REPORT_FILE` is the Semgrep report path
 
+### Utility to integrate the results from the findings API in DefectDojo
+
+To integrate Semgrep Cloud Platform findings into DefectDojo, this utility uses the Semgrep [Findings API](https://semgrep.dev/api/v1/docs/#tag/Finding). It fetches the results, converts them into the JSON format compatible with DefectDojo, and then uploads this formatted report to your specified DefectDojo Product and Engagement.
+
+The primary goal is to insert the Semgrep findings API data into DefectDojo. The users who use Managed Scanning find it difficult as the DefectDojo integration works with Semgrep JSON output file. This is mainly focused on the SAST results. 
+
+You can write a similar script for SCA findings. The [DefectDojo parser](https://github.com/DefectDojo/django-DefectDojo/blob/477583eb674a6a93516a2b794cd9889cb6d7ab81/dojo/tools/semgrep/parser.py) and the sample [Semgrep JSON](https://github.com/DefectDojo/django-DefectDojo/tree/master/unittests/scans/semgrep) files can help understanding the requirements for SCA.
```

Comment:
I probably would merge this with the previous paragraph and frame it as encouraging contributions to expand the functionality of this utility :)